From 661b368802c748095d03fef87bed15442b15110a Mon Sep 17 00:00:00 2001
From: Alvaro Guzman <alvaro.guzman@owasys.com>
Date: Fri, 29 Jul 2022 12:31:46 +0200
Subject: [PATCH] Include resin env

---
 include/env_default.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index 56a8bae3..2653e9b8 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -9,6 +9,8 @@
 
 #include <env_callback.h>
 
+#include <env_resin.h>
+
 #ifdef DEFAULT_ENV_INSTANCE_EMBEDDED
 env_t embedded_environment __UBOOT_ENV_SECTION__(environment) = {
 	ENV_CRC,	/* CRC Sum */
@@ -21,6 +23,7 @@ static char default_environment[] = {
 #else
 const uchar default_environment[] = {
 #endif
+	BALENA_ENV
 #ifndef CONFIG_USE_DEFAULT_ENV_FILE
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
-- 
2.25.1

