From 5cf946b3562d0065d4080df6ee7dd07ee70ed0f0 Mon Sep 17 00:00:00 2001
From: Alvaro Guzman <alvaro.guzman@owasys.com>
Date: Thu, 4 Aug 2022 11:03:07 +0200
Subject: [PATCH 4/4] Removed grammar error

---
 include/configs/imx8mp_owa5x.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mp_owa5x.h b/include/configs/imx8mp_owa5x.h
index c792a30d..6ed8f0a9 100644
--- a/include/configs/imx8mp_owa5x.h
+++ b/include/configs/imx8mp_owa5x.h
@@ -102,7 +102,7 @@
       "fi;" \
       "run nandboot;\0" \
 +	"bootcmd=run BOOT_CMD_BALENA\0" \
-+	"BOOT_CMD_BALENA=ext4load mmc 1:2 ${loadaddr} boot/Image; fatload mmc 1:1 ${fdt_addr} imx8mp-owa5x.dtb; booti ${loadaddr} - ${fdt_addr};\0" \
++	"BOOT_CMD_BALENA=ext4load mmc 1:2 ${loadaddr} boot/Image; fatload mmc 1:1 ${fdt_addr} imx8mp-owa5x.dtb; booti ${loadaddr} - ${fdt_addr}\0" \
 +	"run resin_set_kernel_root;" \
 +	"setenv bootargs ${resin_kernel_root} rootfstype=ext4 rootwait console=$console;" \
 +	"fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${resin_kernel_load_addr} zimage;" \
-- 
2.25.1

