From 7d8ff8f653fe1ce3e8951a1ca0c503c381cd2ecb Mon Sep 17 00:00:00 2001
From: Alvaro Guzman <alvaro.guzman@owasys.com>
Date: Fri, 5 Aug 2022 13:20:30 +0200
Subject: [PATCH 6/6] Modified bootargs to manually boot balenaOS

---
 include/configs/imx8mp_owa5x.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mp_owa5x.h b/include/configs/imx8mp_owa5x.h
index 3f5eabc8..4922140a 100644
--- a/include/configs/imx8mp_owa5x.h
+++ b/include/configs/imx8mp_owa5x.h
@@ -104,7 +104,7 @@
 	"bootcmd=run BOOT_CMD_BALENA\0" \
 	"BOOT_CMD_BALENA=ext4load mmc 1:2 ${loadaddr} boot/Image; fatload mmc 1:1 ${fdt_addr} imx8mp-owa5x.dtb; booti ${loadaddr} - ${fdt_addr};\0" \
 	"run resin_set_kernel_root;" \
-	"setenv bootargs ${resin_kernel_root} rootfstype=ext4 rootwait console=$console;" \
+	"setenv bootargs console=ttymxc0,115200 earlycon=ec_imx6q,0x30860000,115200 root=/dev/mmcblk1p2 rw rootfstype=ext4 rootwait=1;" \
 	"fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${resin_kernel_load_addr} zimage;" \
 	"fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${fdt_addr_r} ${fdtfile};" \
 	"bootz ${resin_kernel_load_addr} - ${fdt_addr_r}\0" \
-- 
2.25.1

